hedera-mirror-rosetta/app/errors/errors.go

```diff
@@ -9,20 +9,32 @@ import (
 // Errors - map of all Errors that this API can return
 var Errors = map[string]*types.Error{
 	BlockNotFound:                New(BlockNotFound, http.StatusNotFound, true),
+	TransactionBuildFailed:       New(TransactionBuildFailed, http.StatusBadRequest, false),
+	TransactionMarshallingFailed: New(TransactionMarshallingFailed, http.StatusBadRequest, false),
 	TransactionNotFound:          New(TransactionNotFound, http.StatusNotFound, true),
+	OnlyOneOperationTypePresent:  New(OnlyOneOperationTypePresent, http.StatusBadRequest, false),
```

Comment:
I think that a better name of the variable might be: "MultipleOperationTypesPresent" or smth like that.
The `OnlyOneOperationTypePresent` sounds to me like there is only one operation type that is present and that's why the error is thrown (the other way around).